rasa/core/exporter.py

```diff
@@ -89,7 +90,10 @@ def publish_events(self) -> int:
                 logger.exception(e)
                 raise PublishingError(current_timestamp)
 
-        self.event_broker.close()
+        if asyncio.iscoroutinefunction(self.event_broker.close):
+            self.event_broker.close()
+        else:
+            await self.event_broker.close()
```

Comment:
that was actually a bug I just recently fixed 🙈 
```suggestion
        if asyncio.iscoroutinefunction(self.event_broker.close):
            await self.event_broker.close()
        else:
            self.event_broker.close()
```